setenv.sh

```diff
@@ -31,6 +32,8 @@ export KDBTAIL=${TORQDATAHOME}/taildir
 # set rlwrap and qcon paths for use in torq.sh qcon flag functions
 export RLWRAP="rlwrap"
 export QCON="qcon"
+export QCMD="taskset -c 0,1 /usr/bin/rlwrap q"
```

Comment:
Are these two changes not specific to our test installation?

Reply:
Yes, this makes the torq5_develop branch easier to install and run out of the box.

Reply:
ok cool. When we release this we need to remember to remove this